packages/create-project/src/helpers/install.ts

```diff
@@ -0,0 +1,108 @@
+import chalk from 'chalk';
+import spawn from 'cross-spawn';
+
+interface InstallArgs {
+  /**
+   * Indicate whether to install packages using Yarn.
+   */
+  useYarn: boolean;
+  /**
+   * Indicate whether there is an active Internet connection.
+   */
+  isOnline: boolean;
+  /**
+   * Indicate whether the given dependencies are devDependencies.
+   */
+  devDependencies?: boolean;
```

Comment:
Maybe call this `saveAsDev` to avoid confusion with the package.json mapping field?